src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -32,6 +32,11 @@
 abstract class DoctrineType extends AbstractType
 {
     /**
+     * @var DoctrineChoiceLoader[]
+     */
+    private static $choiceLoaders = array();
```

Comment:
but this would just force to load them in memory longer (in tests, this will force choice loaders to stay in memory until the end of the PHPUnit process, which implies keeping the whole Doctrine as well and so on), with no real benefit as we won't need to share the same cache between multiple types.